modules/fundamental/src/organization/endpoints/test.rs

```diff
@@ -68,33 +65,11 @@ async fn all_organizations(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
         ]
     );
 
-    Ok(())
-}
-
-#[test_context(TrustifyContext)]
-#[test(actix_web::test)]
-async fn one_organization(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
-    let app = caller(ctx).await?;
+    db.refresh().await?;
```

Comment:
I'm against re-using database instances. We re-create DB instances for a good reason.